source/includes/fact-restart-resume-delay.rst

```diff
@@ -0,0 +1,6 @@
+.. note:: 
+   
+   Starting in ``mongosync`` 1.7.3, ``mongosync`` can take at least two minutes 
+   to respond when you resume or restart a sync operation. This means that 
+   until ``mongosync`` responds, any calls to the :ref:`c2c-api-progress` 
+   endpoint might fail.
```

Comment:
[Question]

Are failed `/progress` calls safe to retry? Should we specify?

Reply:
I'll ask for clarification in external review on this!